README.md

```diff
@@ -23,6 +23,7 @@ Options:
   -j, --json=JSON      Pass variables as JSON (dictionary string or file path)
   -p, --pair=PAIR      Pass variable as key=value pair (multiple values allowed)
       --query=QUERY    Pass variables as URL query string
+      --env            Treat environment variables as input data
```

Comment:
Could you put this right below `--dir`, and also change the order in the other files to match? The options (aside from the help/version stuff) are meant to be listed alphabetically